\documentclass[12pt]{article}
\usepackage[hcentering,bindingoffset=20mm]{geometry}
\usepackage{placeins}
\usepackage[numbib]{tocbibind}
\usepackage{rotating}
\usepackage[square,sort,comma,numbers]{natbib}
\usepackage{graphicx}
\usepackage{tabularx}
\linespread{1.3}
\usepackage{gensymb}
\usepackage{longtable}
\usepackage{lscape}
\usepackage{url}
\addtolength{\textwidth}{2cm}
\addtolength{\hoffset}{-1cm}
\addtolength{\textheight}{2cm}
\addtolength{\voffset}{-1cm}
\setlength{\parindent}{0pt}
\title{Development of semi-quantitative PCR assays for the detection and enumeration of \emph{Gambierdiscus lapillus} and \emph{Gambierdiscus polynesiensis} (Gonyaulacales, Dinophyces) at the Great Barrier Reef, Australia.}
\author{me}
\date{}
\begin{document}
\maketitle
\paragraph{}Anna Liza Kretzschmar\\
Climate Change Cluster (C3), University of Technology Sydney, Ultimo, 2007 NSW, Australia, anna.kretzschmar@uts.edu.au
\paragraph{}Arjun Verma \\
Climate Change Cluster (C3), University of Technology Sydney, Ultimo, 2007 NSW, Australia
\paragraph{}Gurjeet Kohli\\
Climate Change Cluster (C3), University of Technology Sydney, Ultimo, 2007 NSW, Australia
\paragraph{}Shauna Murray\\
Climate Change Cluster (C3), University of Technology Sydney, Ultimo, 2007 NSW, Australia
\newpage
\section{Abstract}
Ciguatera fish poisoning is a potentially fatal illness contracted through ingestion of seafood containing analogues of ciguatoxins. It is prevalent in tropical regions worldwide, including in Australia. Ciguatoxins are produced by some species of the genus \emph{Gambierdiscus}. Rapid \emph{Gambierdiscus} species identification through quantitative PCR, along with the identification of toxicity of species, is requored for monitoring strategies. In this study we developed a rapid qPCR assay to quantify the species Gambierdiscus lapillus, which is found in Great Barrier Reef region of northern Australia, and produces an uncharacterized analogue of maitotoxin, and also possible ciguatoxin-like compounds. Here, we describe the specificity and efficiency of the method. We tested this assay on samples from 33 sites around Heron Island in the southern Great Barrier Reef. We explore the distribution of \emph{G. lapillus} across spatial replicates and three macroalgal hosts. 


\newpage
\section{Introduction}
%The genus Gambierdiscus and its morphological diversity, and why qPCR is needed to detect species.
Benthic dinoflagellates of the genus \emph{Gambierdiscus} Adachi \& Fukuyo produce neurotoxic ciguatoxins (CTX) which bioaccumulate in fish and other marine life, causing ciguatera fish poisoning (CFP) \citep{chinain1997intraspecific,holmes1998gambierdiscus}. 

The genus was discovered in 1977 with \emph{G. toxicus} Adachi \& Fukuyo \citep{adachi1979thecal} and the taxon was considered monotypic till 1995, when \emph{G. belizeanus} Faust was discovered \citep{faust1995observation}. To date the genus has expanded to 13 described species and 6 ribo/species types
 \citep{smith2016new,fraga2016gambierdiscus,litaker2010global,adachi1979thecal,faust1995observation,chinain1999morphology,litaker2009taxonomy,nishimura2014morphology,kretzschmar2016characterization,fraga2011gambierdiscus,xu2014distribution,fraga2014genus} .
Major revision in \emph{Gambierdiscus} species resolution was undertaken by Litaker et al. (2009). Reports of \emph{Gambierdiscus} spp. identified based on morphology alone prior to this revision need to be considered with caution as intra-species variation and inter-species similarities can cause misidentification \citep{bravo2014cellular,kretzschmar2016characterization,kohli2014high}. Hence molecular genetic tools are essential to determine the distribution and abundance of  \textit{Gambierdiscus} species  and assessment of ciguatera risk \citep{kohli2014high,kretzschmar2016characterization}. \\

%Toxins of Gambierdiscus. How to measure. Which species produce which toxins. Why this is important to know.

\emph{Gambierdiscus} spp. produce a suite of different polyketide toxins - CTX, maitotoxin (MTX), gambierone, gambieric acid and gambierol have been characterised to date \citep{satake1993gambierol,nagai1992gambieric,rodriguez2015gambierone,murata1993structure,murata1989structures}. While any of these can contribute to toxicity, only CTX has been clearly found to cause CFP \citep{chinain1997intraspecific,holmes1998gambierdiscus}. It is not yet well understood which \textit{Gambierdiscus} species produce CTXs, and many different assays have been used to determine CTX toxicity \citep{globalcig}. Bioassays, such as mouse bioassays and neuroblastoma cell-line bioassays are good indicators of the toxicity of an organism, however species/strain specific toxin profiles needs to be elucidated with LC-MS/MS in order to characterize individual toxin congeners \citep{diogened2014chemistry}. \textit{Gambierdiscus polynesiensis} Chinain \& Faust is the only species of \emph{Gambierdiscus} whose production of CTX congeners has been consistently verified by LC-MS/MS, and is thought to be the principal cause of CFP in the Pacific region \citep{chinain2010growth,rhodes2014production}. The production of CTX-like compounds \emph{Gambierdiscus lapillus} is currently being investigated, as an uncharacterised CTX-like peak has been detected using LC-MS/MS \cite{kretzschmar2016characterization}. Determining the toxin profile for species of \textit{Gambierdiscus} requires toxin standards for comparative peak analysis; however these are currently not commercially available. Therefore, progress in determining the toxins produced by species of \emph{Gambierdiscus} has been comparatively slow.\\

%CFP 
The diagnosis and report of CFP is difficult due to the breadth of gastrointestinal, cardiovascular and neurological presentation of CFP and relies on specific training of the physician \citep{sims1987theoretical}. 
To prevent outbreaks, monitoring programs are important. 

CFP was determined  to be  a neglected tropical disease by a panel of experts co-ordinated by the UNESCO, and a global ciguatera strategy was developed \cite{globalcig}. The global ciguatera strategy involves investigating species of the genus \emph{Gambierdiscus}, elucidating which species produce CTX through LC-MS/MS and developing efficient and reliable molecular monitoring tools for the species of interest \citep{globalcig}. Quantitative PCR (qPCR) is an adequate target for molecular screening tool development as it can give species specific and quantitative results from community DNA extracts. This requires primer development and verification \citep{globalcig}. \\

%Which qPCR assays have been developed to date, why more are necessary
For qPCR based species identification of \emph{Gambierdiscus}, assays have been developed either using SYBR Green of TaqMan probes \cite{vandersea2012development,nishimura2016quantitative}. The former relies on target specific primer design and flourometric meassurement of the chelating agent SYBR Green, which is directly proportional to the amount of PCR product present after each cycle. The latter relies on the release of a flourophore upon incoropration of a target specific probe, indexed to the PCR product present per cycle \citep{smith2009advantages}. Currently the qPCR assays available for \emph{Gambierdiscus} identification are as follows: \emph{G. belizeanus}, \emph{G. caribaeus} Vandersea, Litaker, Faust, Kibler, Holland \& Tester, \emph{G. carpenteri} Vandersea, Litaker, Faust, Kibler, Holland \& Tester, \emph{G. carolinianus} Vandersea, Litaker, Faust, Kibler, Holland \& Tester, \emph{Gambierdiscus} sp. ribotype 2 and \emph{G. (Fukoyoa) ruetzleri} (Faust, Litaker, Vandersea, Kibler, Holland \& Tester) G\`omez, Qiu, Lopes \& Lin utilising SYBR Green \citep{vandersea2012development}. Nishimura et al. (2016) developed TaqMan probes for \emph{G. australes} Chinian \& Faust, \emph{G. scabrosus} Nishimura, Sato \& Adachi, \emph{Gambierdiscus} sp. type 2, \emph{Gambierdiscus} sp. type 3 and \emph{G. (Fukoyoa)} cf. \emph{yasumotoi} (Holmes) G\`omez, Qiu, Lopes \& Lin \citep{nishimura2016quantitative}. This constitutes 6 out of 13 verified species specific qPCR primer sets for described \emph{Gambierdiscus} spp. and 3 out of 6 undescribed \emph{Gambierdiscus} sp. types/ribotypes , as well as 2 out of 3 qPCR primer sets for species of the genus \emph{Fukoyoa} (seceded from the genus \emph{Gambierdiscus} in 2015 \citep{gomez2015fukuyoa}) \textit{Fukoyoa} spp. are of interest as MTX producers, as the involvement of that toxin in CFP has not been resolved \citep{kohli2014feeding}. \\

%Australia
Australia has CFP endemic areas, most notably the Great Barrier Reef, yet the cases are highly under reported with a predicted rate of 20\% \citep{lewis2006ciguatera}. The country's coastline is expansive and sites sampled for the presence of \emph{Gambierdiscus} spp. are sparse \cite{kretzschmar2016characterization}. The 4 species of \emph{Gambierdiscus} that have been identified from Queensland and New South Wales are as follows: \emph{G. belizeanus} \citep{murray2014molecular}, \emph{G. carpenteri} \citep{kohli2014high}, \emph{G. lapillus} \cite{kretzschmar2016characterization} and \emph{G. toxicus} \citep{hallegraeff2010algae}, as well as \emph{F. yasumotoi} \citep{murray2014molecular}. Using pyrosequencing, \textit{Gambierdiscus} was identified \emph{Gambierdiscus} to the genus level in Western Australia \citep{kohli2014cob} , indicating that this is a coastline that should be examined further for CFP risk. Despite the prevalence of CFP in Australia, a species of \textit{Gambierdiscus} that consistently produces ciguatoxins has not yet been identified.
qPCR primers that can be used for identifying and monitoring Australia have been developed for \emph{G. belizeanus}, \emph{G. carpenteri} and \emph{F. yasumotoi} \citep{nishimura2016quantitative,vandersea2012development}. \\ 
% buuut Aussie one may be F. paulensis from mediterranean paper

%aim
The aim of this study was to develop qPCR primers assays to detect the two species \emph{G. lapillus} and \emph{G. polynesiensis}. The latter is a species found in the Pacific that produces a range of CTXs congeners (CTX-3B, CTX-3C, CTX-4A, CTX-4B), identified by LC-MS \cite{rhodes2014production}, and would be a plausible causative agent of CFP in Australia. The former is a recently identified species from the Great Barrier Reef whose toxin profile did not include any of the known CTXs but displayed an unusual peak in the CTX phase of the LC-MS which makes it a potentially important candidate to monitor. Neuroblastoma cell assay was conducted to investigate CTX specific toxicity and draw inferences for environmental relevance.
\newpage
\section{Materials and methods}
\subsection{Clonal strains and culturing conditions}
Three strains of \emph{G. lapillus} and two strains of \emph{G. polynesiensis} were isolated from Heron Island, Australia, and Rarotonga, Cook Islands, respectively (table ~\ref{tbl:StrainTable}). The cultures were maintained F$\bullet$-10 medium at 27 $^{\circ}$C, 60mol$\bullet$-m$^{2}$ $\bullet$-s light in 12hr:12hr light to dark cycles.
\FloatBarrier
\begin{table}
\caption{List of \emph{Gambierdiscus} clonal strains used for the qPCR assay.}
\label{tbl:StrainTable}
\begin{tabular}{ | p{2cm} | p{2cm} | p{2cm}| p{3cm} | p{3cm} | p{2cm} | }
\hline
\textbf{Species} & \textbf{Collection site} & \textbf{Collection date} &\textbf{Latitude} & \textbf{Longitude} & \textbf{Strain name} \\
\hline
\emph{G. lapillus} &Heron Island, Australia &July 2014 &23$^{\circ}$ 4420' S&151$^{\circ}$ 9140' E & HG4 \\
\hline
&&&&& HG6\\
\hline
&&&& &HG7\\
\hline
&&&& &HG26\\
\hline
\emph{G. polynesiensis}&Rarotonga, Cook Islands&November 2014 &21$^{\circ}$ 2486' S&159$^{\circ}$ 7286' W & CG14 \\
\hline
&&&&&CG15\\
\hline
\end{tabular}
\end{table}

\subsection{DNA extraction and species specific primer design}
Genomic DNA was extracted using the CTAB method \citep{zhou1999analysis}. Purity and concentration of the extract was measured by Nanodrop (Nanodrop2000, Thermo Scientific), the integrity was visualised on 1\% agarose gel.
Unique primer sets were designed for the small-subunit (SSU) rDNA region of \emph{G. lapillus}, as reported by Kretzschmar et al., and \emph{G. polynesiensis}, as available in GenBank reference database. The target sequences were aligned against sequences of all other \emph{Gambierdiscus}spp. from GenBank reference database, with the MUSCLE algorithm (maximum of 8 iterations) \citep{edgar2004muscle} used through the Geneious software, version 8.1.7 \citep{kearse2012geneious}. Unique sites were determined manually (Table ~\ref{tbl:PrimerTable}). Primers were synthesised by Integrated DNA Technologies (IA, USA).
Primer sets were tested systematically for secondary product formation for all 3 strains of \emph{G. lapillus} and 2 strains of \emph{G. polynesiensis} (Table ~\ref{tbl:StrainTable}) via standard PCR in 25$\mu$l mixture in PCR tubes. The mixture contained 0.6 $\mu$M forward and reverse primer, 0.4 $\mu$M BSA, 2 - 20 ng DNA, 12.5 $\mu$l 2xEconoTaq (Lucigen) and 7.5 $\mu$l PCR grade water.
The PCR cycling comprised of an initial 10 min step at 94 $^{\circ}$C, followed by 30 cycles of denaturing at 94 $^{\circ}$C for 30 sec, annealing at 60 $^{\circ}$C for 30 sec and extension at 72 $^{\circ}$C for 1 min, finalised with 3 minutes of extension at 72 $^{\circ}$C. Products were visualised on 1\% agarose gel.
\FloatBarrier
\begin{table}
\caption{List of species specific qPCR primer sets for SSU rDNA.}
\label{tbl:PrimerTable}
\begin{tabular}{ | p{2cm} | p{2cm} | p{2cm} | p{2cm} | p{7cm} | }
\hline
\textbf{DNA target} & \textbf{Amplicon size} & \textbf{Primer name} & \textbf{Synthesis direction of primer} & \textbf{Sequence (5'-3')} \\
\hline
\emph{G. lapillus} & &qGlapSSU2F & Forward & \\
\hline
& &qGlapSSU2R & Reverse & \\
\hline
\emph{G. polynesiensis}& &qGpolySSU2F& Forward & \\
\hline
& &qGpolySSU2R & Reverse & \\
\hline
\end{tabular}
\end{table}
\subsection{Evaluation of primer specificity}
To verify primer set specificity as listed in Table ~\ref{tbl:PrimerTable}, DNA was extracted via CTAB from \emph{G. belizeanus} (CCMP401), \emph{G. australes} (CCMP1650 and CG61) and \emph{G. pacificus} (CAWD149). \emph{G. cheloniae} (CAWD232) DNA was extracted using a PowerSoilâ„¢ DNA isolation kit (Mo Bio Inc., CA, USA) by Dr. Kirsty Smith, Cawthron Institute, New Zealand. Cross reactivity for \emph{G. scabrosus} (strains?) DNA was extracted using DNeasy Plant Mini Kit (Quiagen, Tokyo, Japan) according to the manufacturer's protocol, by Dr. Nishimura, Kochi University, Japan. For all extracted samples, the presence and integrity of genomic DNA was assessed on 1\% agarose gel. Primer sets designed for \emph{G. lapillus} and \emph{G. polynesiensis} were tested for cross-reactivity against all other \emph{Gambierdiscus} spp. available via PCR. PCR amplicons were visually assessed on 1\% agarose gel.


\subsection{Evaluation of primer sensitivity}
The qPCR reaction mixture contained 10 $\mu$l SYBR Select Master Mix (Thermo Fisher Scientific), 7 $\mu$l MilliQ water, 0.5 $\mu$M forward and reverse primers and 2 - 20 ng DNA template, for a final volume of 20 $\mu$l.
\subsection{Calibration curve construction}
Standard curves were constructed via two methods: (1) synthetic gene fragment based (henceforth referred to as gene based calibration) and (2) cell based calibration curves. The former was constructed from a 10-fold serial dilution of a synthesised fragment containing the SSU target sequence, forward and reverse primer sites and 50bp flanking both primer sites matching sequencing results. Cell-based standard curves were constructed using 10-fold dilutions of gDNA extract of known cell concentrations.
\subsubsection{Gene based calibration curve}
A synthetic gBlocks gene fragment spanning the target SSU sequence, primer sites and 50bp on either end was synthesised for both \emph{G. lapillus} and \emph{G. polynesiensis}.
\subsubsection{Cell based calibration curve}
Three cultures of \emph{G. lapillus} and two cultures of \emph{G. polynesiensis} (Table ~\ref{tbl:StrainTable}) were used to construct cell based standard curves. Cells were counted microscopically () with a Sedqick Rafter Counting Chamber. DNA was extracted using xx kit, as per the manufacturer's instructions. The gDNA extracts were 10-fold serial diluted.
%\subsection{Quantification of SSU rDNA copy number per cell of \emph{G. lapullis} and \emph{G. polynesiensis}}
\subsection{Toxicity of \emph{G. lapillus}}
- if N2A assays work out
\FloatBarrier
\begin{table}
\caption{\emph{Gambierdiscus} spp. analysed by N2A assay including strain ID and cell counts.}
\label{tbl:N2ATable}
\begin{tabular}{ | p{4cm} | p{3cm} | p{2cm} | }
\hline
\textbf{Species} & \textbf{Strain ID}& \textbf{Cell count} \\
\hline
\emph{G. lapillus}&HG4&388,020\\
\hline
&HG7&1,533,460\\
\hline
&HG26&113,280\\
\hline
\emph{G. polynesiensis}&CG14&438,480\\
\hline
&CG15&675,840\\
\hline
\end{tabular}
\end{table}
\FloatBarrier

\subsection{Screening environmental samples for \emph{G. lapullis} and \emph{G. polynesiensis} abundance}
Around Heron Island and Heron Reef (Fig. ~\ref{fig:samplesites}) 33 sites were sampled in October 2015, in spatial replicates (A, B, C; ~\ref{tbl:MacroalgaeTable}) within a 2m radius. Macroalgae targeted were \textit{Chnoospora} sp., \textit{Padina} sp. and \textit{Saragassum} sp. where ~200g were taken, epiphytes were dislodged from macroalagae and preserved in 10ml RNAlater till DNA extraction.
Plankton nets were dragged through the current (samples 34 - 38; table ~\ref{tbl:NetTable}) for five minutes, then processed as macroalgal samples.

\FloatBarrier 
\begin{figure} 
\includegraphics[scale=1.5]{Heron_sample-map2.png} 
\caption{(A) shows Australia, with the position of Heron Island (grey circle); (B) shows Heron Island including surrounding reefs; (C) shows the sampling sites around Heron Island.} 
\label{fig:samplesites}
\end{figure} 
\FloatBarrier

\newpage
\section{Results}
\subsection{Evaluation of primer specificity}
The primers were specific to their target species for both \emph{G. lapillus} and \emph{G. polynesiensis} (table ~\ref{tbl:CrossreactTable}). The negative PCR  amplification is visualised in Fig. ~\ref{fig:lapgel} and Fig. xx respectively.
\FloatBarrier 
\begin{figure} 
\includegraphics[scale=.6]{Gel_primer-spec.png} 
\caption{Gel electrophoresis visualisation of PCR using qGlapSSU2F and qGlapSSU2R primer sets on \emph{G. lapillus}, \emph{G. belizeanus}, \emph{G. cheloniae}, \emph{G. pacificus} and \emph{G. scabrosus} gDNA as templates.} 
\label{fig:lapgel}
\end{figure} 
\FloatBarrier

\FloatBarrier
\begin{table}
\caption{Cross-reactivity of qPCR primer sets.}
\label{tbl:CrossreactTable}
\begin{tabular}{ | p{4cm} | p{3cm} | p{2cm} | p{2.5cm} | p{2.5cm} | }
\hline
\textbf{Template} & \textbf{Strain name} & \textbf{gDNA gel band} & \textbf{GlapSSU2F-GlapSSU2R} & \textbf{GpolySSUF-GpolySSUR} \\
\hline
\emph{G. australes} & CCMP1650 &+&-&- \\
\hline
& CG61 &+&-&- \\
\hline
\emph{G. belizeanus}&CCMP401&+&-&-\\
\hline
\emph{G. lapillus}&HG1&+&+&-\\
\hline
&HG4&+&+&-\\
\hline
&HG6&+&+&-\\
\hline
&HG7&+&+&-\\
\hline
&HG26&+&+&-\\
\hline
\emph{G. pacificus}&CAWD149&+&-&-\\
\hline
\emph{G. polynesiensis}&CG14&+&-&+\\
\hline
&CG15&+&-&+\\
\hline
\emph{G. scabrosus}&&+&-&-\\
\hline
\end{tabular}
\end{table}
\FloatBarrier
 
\subsection{Evaluation of primer sensitivity}
-figs qPCR std curves
\FloatBarrier
\begin{figure}
%\includegraphics[scale=.22]{MH_variability.jpg}
\caption{qPCR standard curves of \emph{G. lapillus} strains (A) HG4 and (B) HG7. Error bars represent the deviation of technical replicates during reactions.}
\label{fig:lapiStd}
\end{figure}
\FloatBarrier
\FloatBarrier
\begin{figure}
%\includegraphics[scale=.22]{MH_variability.jpg}
\caption{qPCR standard curves of \emph{G. polynesiensis} strains (A) CG14 and (B) CG15. Error bars represent the deviation of technical replicates during reactions.}
\label{fig:polyStd}
\end{figure}
\FloatBarrier
-Ct of known gBlocks gives copy number - extrapolate those to SSU rDNA copies for cell. can compare to
\subsection{Quantification of SSU rDNA copy number per cell of \emph{G. lapullis} and \emph{G. polynesiensis}}
-gblocks calc
\subsection{Screening environmental samples for \emph{G. lapullis} and \emph{G. polynesiensis} abundance}
- map of Heron island and sampling sites
- not sure how to label Table ~\ref{tbl:MacroalgaeTable} biological rep section, will change that table once I get clarity on that.
\FloatBarrier
\begin{longtable}{ | p{1cm} | p{1cm} | p{3cm} | p{4cm} | p{4cm} | }
\caption{Screening of macroalgal samples of \emph{G. lapillus} and \emph{G. polynesiensis}and cell density estimates via qPCR. \emph{G. lapillus} cells numbers were modelled on the type strain HG7. \emph{G. polynesiensis} cell numbers were modelled on the strain CG14. N/D denotes not detected; N/A denotes not attempted due to loss of sample.}\\
\hline
\label{tbl:MacroalgaeTable}
\textbf{Sample ID}&\textbf{Spatial rep}&\textbf{Macroalgal substrate}&\textbf{\textit{G. lapillus} cell}&\textbf{\textit{G. polynesiensis} SSU rDNA copy number }\\
\hline
1&A&\emph{Padina} sp.&N/D&N/D\\
\hline
1&B&\emph{Sargassum} sp.&10.55
&N/D\\
\hline
1&C&\emph{Padina} sp.&2.75
&N/D\\
\hline
2&A&\emph{Padina} sp.&N/D&N/D\\
\hline
2&B&\emph{Padina} sp. \& \emph{Chnoospora sp.}&4.33
&N/D\\
\hline
2&C&\emph{Padina} sp.&4.27
&N/D\\
\hline
3&A&\emph{Padina} sp.&0.62
&N/D\\
\hline
3&B&\emph{Padina} sp.&N/D&N/D\\
\hline
3&C&\emph{Padina} sp.&1.99
&N/D\\
\hline
4&A&\emph{Padina} sp. \& \emph{Chnoospora sp.}&6.13
&N/D\\
\hline
4&B&\emph{Chnoospora sp.}&0.62
&N/D\\
\hline
4&C&\emph{Padina} sp. \& \emph{Chnoospora sp.}&N/D&N/D\\
\hline
5&A&\emph{Padina} sp.&N/D&N/D\\
\hline
5&B&\emph{Padina} sp.&2.41
&N/D\\
\hline
5&C&\emph{Padina} sp.&3.81
&N/D\\
\hline
6&A&\emph{Chnoospora sp.}&1.12
&N/D\\
\hline
6&B&\emph{Padina} sp.&1.65
&N/D\\
\hline
6&C&\emph{Padina} sp.&N/D&N/D\\
\hline
7&A&\emph{Padina} sp.&9.35
&N/D\\
\hline
7&B&\emph{Padina} sp.&N/D&N/D\\
\hline
7&C&\emph{Padina} sp.&N/D&N/D\\
\hline
8&A&\emph{Chnoospora sp.}&N/D&N/D\\
\hline
8&B&\emph{Padina} sp.&1.67
&N/D\\
\hline
8&C&\emph{Padina} sp. \& \emph{Chnoospora sp.}&3.61
&N/D\\
\hline
9&A&\emph{Padina} sp. \& \emph{Chnoospora sp.}&N/D&N/D\\
\hline
9&B&\emph{Padina} sp.&1.69
&N/D\\
\hline
9&C&\emph{Padina} sp.&1.92
&N/D\\
\hline
10&A&\emph{Padina} sp. \& \emph{Chnoospora sp.}&12.6
&N/D\\
\hline
10&B&\emph{Padina} sp. \& \emph{Chnoospora sp.}&4.01
&N/D\\
\hline
10&C&\emph{Padina} sp.&N/D&N/D\\
\hline
11&A&\emph{Padina} sp. \& \emph{Sargassum} sp.&N/D&N/D\\
\hline
11&B&\emph{Padina} sp. \& \emph{Sargassum} sp.&0.26
&N/D\\
\hline
11&C&\emph{Padina} sp. \& \emph{Sargassum} sp.&1.29
&N/D\\
\hline
12&A&\emph{Chnoospora sp.}&N/D&N/D\\
\hline
12&B&\emph{Chnoospora sp.}&17.09
&N/D\\
\hline
12&C&\emph{Chnoospora sp.}&4.27
&N/D\\
\hline
13&A&\emph{Chnoospora sp.}&N/D&N/D\\
\hline
13&B&\emph{Padina} sp. \& \emph{Chnoospora sp.}&49.51
&N/D\\
\hline
13&C&\emph{Padina} sp. \& \emph{Chnoospora sp.}&18.58
&N/D\\
\hline
14&A&\emph{Padina} sp. \& \emph{Chnoospora sp.}&0.91
&N/D\\
\hline
14&B&\emph{Padina} sp. \& \emph{Chnoospora sp.}&N/D&N/D\\
\hline
14&C&\emph{Chnoospora sp.}&5.95
&N/D\\
\hline
15&A&\emph{Padina} sp. \& \emph{Chnoospora sp.}&2.01
&N/D\\
\hline
15&B&\emph{Chnoospora sp.}&4.89
&N/D\\
\hline
15&C&\emph{Chnoospora sp.}&N/D&N/D\\
\hline
16&A&\emph{Chnoospora sp.}&6.70&N/D\\
\hline
16&B&\emph{Chnoospora sp.}&8.83
&N/D\\
\hline
16&C&\emph{Chnoospora sp.}&3.08
&N/D\\
\hline
17&A&\emph{Chnoospora sp.}&2.58
&N/D\\
\hline
17&B&\emph{Chnoospora sp.}&9.39
&N/D\\
\hline
17&C&\emph{Padina} sp. \& \emph{Chnoospora sp.}&N/D&N/D\\
\hline
18&A&\emph{Chnoospora sp.}&0.02
&N/D\\
\hline
18&B&\emph{Chnoospora sp.}&N/D&N/D\\
\hline
18&C&\emph{Chnoospora sp.}&9.24
&N/D\\
\hline
19&A&\emph{Chnoospora sp.}&5.27
&N/D\\
\hline
19&B&\emph{Padina} sp. \& \emph{Chnoospora sp.}&48.46
&N/D\\
\hline
19&C&\emph{Padina} sp. \& \emph{Chnoospora sp.}&2.71
&N/D\\
\hline
20&A&\emph{Padina} sp. \& \emph{Chnoospora sp.}&2.81
&N/D\\
\hline
20&B&\emph{Padina} sp. \& \emph{Chnoospora sp.}&10.26
&N/D\\
\hline
20&C&\emph{Chnoospora sp.}&N/D&N/D\\
\hline
21&A&\emph{Padina} sp. \& \emph{Chnoospora sp.}&5.50
&N/D\\
\hline
21&B&\emph{Chnoospora sp.}&1.23
&N/D\\
\hline
21&C&\emph{Padina} sp.&10.32
&N/D\\
\hline
22&A&\emph{Chnoospora sp.}&N/D&N/D\\
\hline
22&B&\emph{Chnoospora sp.}&37.68
&N/D\\
\hline
22&C&\emph{Chnoospora sp.}&5.57
&N/D\\
\hline
23&A&\emph{Chnoospora sp.}&7.86
&N/D\\
\hline
23&B&\emph{Chnoospora sp.}&N/D&N/D\\
\hline
23&C&\emph{Chnoospora sp.}&3.14
&N/D\\
\hline
24&A&\emph{Padina} sp. \& \emph{Chnoospora sp.}&0.72
&N/D\\
\hline
24&B&\emph{Padina} sp. \& \emph{Chnoospora sp.}&0.58
&N/D\\
\hline
24&C&\emph{Padina} sp.&3.68
&N/D\\
\hline
25&A&\emph{Padina} sp.&N/D&N/D\\
\hline
25&B&\emph{Padina} sp.&N/D&N/D\\
\hline
25&C&\emph{Padina} sp.&N/D&N/D\\
\hline
26&A&\emph{Sargassum} sp.&N/D&N/D\\
\hline
26&B&\emph{Sargassum} sp.&0.19
&N/D\\
\hline
26&C&\emph{Sargassum} sp.&0.18
&N/D\\
\hline
27&A&\emph{Sargassum} sp.&N/D&N/D\\
\hline
27&B&\emph{Sargassum} sp.&2.11
&N/D\\
\hline
27&C&\emph{Sargassum} sp.&2.05
&N/D\\
\hline
28&A&\emph{Padina} sp.&7.17
&N/D\\
\hline
28&B&\emph{Padina} sp.&2.67
&N/D\\
\hline
28&C&\emph{Padina} sp.&8.64
&N/D\\
\hline
29&A&\emph{Chnoospora sp.}&1.24
&N/D\\
\hline
29&B&\emph{Chnoospora sp.}&5.90
&N/D\\
\hline
29&C&\emph{Chnoospora sp.}&N/D&N/D\\
\hline
30&A&\emph{Padina} sp. \& \emph{Sargassum} sp.&4.09
&N/D\\
\hline
30&B&\emph{Padina} sp. \& \emph{Sargassum} sp.&2.58
&N/D\\
\hline
30&C&\emph{Padina} sp. \& \emph{Sargassum} sp.&N/D&N/D\\
\hline
31&A&\emph{Sargassum} sp.&1.91
&N/D\\
\hline
31&B&\emph{Sargassum} sp.&2.90
&N/D\\
\hline
31&C&\emph{Sargassum} sp.&3.97
&N/D\\
\hline
32&A&\emph{Padina} sp. \& \emph{Sargassum} sp.&2.24
&N/D\\
\hline
32&B&\emph{Chnoospora sp.} \& \emph{Sargassum} sp.&1.36
&N/D\\
\hline
32&C&\emph{Padina} sp. \& \emph{Sargassum} sp.&2.00
&N/D\\
\hline
33&A&\emph{Padina} sp.&4.73
&N/D\\
\hline
33&B&\emph{Padina} sp. \& \emph{Chnoospora sp.}&0.07
&N/D\\
\hline
33&C&\emph{Padina} sp. \& \emph{Chnoospora sp.}&2.44
&N/D\\
\hline
\end{longtable}
\FloatBarrier
\FloatBarrier
\begin{table}
\caption{Screening of tow net samples for free floating \emph{G. lapillus} and \emph{G. polynesiensis} and cell density estimates via qPCR.}
\label{tbl:NetTable}
\begin{tabular}{ | p{4cm} | p{4cm} |p{4cm} | p{4cm} | }
\hline
\textbf{Sample ID}&\textbf{Location}&\textbf{\emph{G. lapillus}cell density}&\textbf{\emph{G. polynesiensis} cell density. N/D denotes not detected.}\\
\hline
34&Shark Bay A?&N/D&N/D\\
\hline
35&Shark bay B&N/D&N/D\\
\hline
36&Shark Bay D&1.24
&N/D\\
\hline
37&North beach D&0.70
&N/D\\
\hline
38&Jetty&N/D
&N/D\\
\hline
\end{tabular}
\end{table}
\FloatBarrier

\FloatBarrier 
\begin{figure} 
\includegraphics[scale=1.5]{Heron-positive-negative-samplingsites.png} 
\caption{Shows the detection of \emph{G. lapillus} at the sampling sites around Heron Island. The spatial replicates for each site are set up as shown in (A); the sites in (B) linked to numbering in Fig. ~\ref{fig:samplesites} where positive (green) and negative (red) as per Table ~\ref{tbl:MacroalgaeTable}.} 
\label{fig:envposneg}
\end{figure} 
\FloatBarrier

\FloatBarrier 
\begin{figure} 
\includegraphics[scale=.3]{HG7-env.png} 
\caption{Detection of \emph{G. lapillus} per spatial replicate at each sampling site, cell numbers as normalised to HG7 standard curve (Fig. ~\ref{fig:lapiStd}B). Spatial replicates coloured as per macroalgal substrate,\ where \emph{Chnoospora} sp. are green, \emph{Sargassum} sp. are blue, \emph{Padina} sp. are red and mixed macroalgal substrates are yellow (see table ~\ref{tbl:MacroalgaeTable}).} 
\label{fig:envHG7}
\end{figure} 
\FloatBarrier

\newpage
\section{Discussion}
%aim of study, ways to achieve aim, results, extrapolate for accurate tool development
The authors set out to develop rapid molecular screening tools for \emph{G. lapillus} and \emph{G. polynesiensis}. Secondary aims were to assess the toxicity of \emph{G. lapillus} and query the presence of \emph{G. polynesiensis} at Heron Island in the ongoing quest to find the causative species for CFP in Australia. To achieve these goals, species specific PCR primers with high specificity and sensitivity were developed; SSU copy number for two strains per species were elucidated; and efficacy of the primer sets for environmental screening. The cross-reactivity of primers designed in this study showed high specificity for both \emph{G. lapillus} and \emph{G. polynesiensis} as compared to other \emph{Gambierdiscus} sp. (Fig. ~\ref{fig:lapgel}). Standard curves were constructed for two strains of each target species and the primers showed high linearity and amplification efficiency (Fig. ~\ref{fig:lapiStd}, Fig. ~\ref{fig:polyStd}). Hence these primer sets can accurately and reproducible molecular tool to enumerate \emph{G. lapillus} and \emph{G. polynesiensis} cells.\\


%nishimura 16 no diff beteen cell growth phases and SSU # copy 
The difference in SSU copy number between strains in both \emph{G. lapillus} (HG4 , HG7) and \emph{G. polynesiensis} (CG14 , CG15) could be due to a number of procedural problems: over or under estimation from cell counts due to non-uniform cell suspension of \emph{Gambierdiscus}; loss of cells during the standard curve DNA extraction; or change of in Ct values between runs. Precautions were taken to address these possibilities. The aggregation of \emph{Gambierdiscus} spp. has been reported by CITE and is commonly observed by anybody that cultures \emph{Gambierdiscus}. Hence cultures were heavily agitated before sample collection for counting. To account for DNA loss during extraction, the gBlocks standards were extracted by the same procedure as the cell based standards, modelled on the experimental procedure by Kon et al. (2015) \cite{kon2015spatial}. To avoid error from run based differences in Ct values, the same machine was used for all runs and sub samples of the cell standards included in the gBlocks run (as well as the environmental screening) to assess the experimental Ct value compared to a known Ct value. 
Nishimura et al. (2016) assessed the SSU copy variation of \emph{G. scabrosus} at different growth phases and found no significant differences. This trend has also been recorded for \emph{Alexandrium catanella} and \emph{Alexandrium taylori}, though the latter also showed growth phase dependent variation in SSU copy numbers \cite{galluzzi2010analysis}.
This indicates that the strain variation of \emph{G. lapillus} and \emph{G. polynesiensis} SSU copy numbers is likely not due to difference in growth phase when harvesting respective strains, though the consistency of \emph{Gambierdiscus} not displaying species specific variability as shown in \emph{Alexandrium} should be verified.
Due to these precautions, the authors are reasonably confident that the intra strain variation in SSU copies is a reflection of reality rather than due to procedural error or growth phase variation. Subsequently, the estimation of cells in environmental samples by comparison to a cell based standard will likely be slightly inaccurate. However the discrepancy would be minor, so qPCR enumeration is still considered accurate and an adequate screening tool for monitoring. 

%^ negligeable dif in SSU# for cultured and wild type for 4 phylotypes therefore culture based std is adequate predictor for environmental cell numbers ~  detection limits --> Further, examining the difference in SSU copy numbers between 

%compare between strains tested

% should use RE

% do gblocks calc and compare total SSU copy # to Nishimura numbers and Vandersea, weigh in on methodology discussion

%incluude the estimated SSU rDBA copy calculations by Godhe '08 \cite{godhe2008quantification} and Sun '03 \cite{sun2003geometric} then compare

%semi quant nature of screening
The environmental screening results for \emph{G. lapillus} of this study were termed semi-quantitative as the environmental cell numbers detected (table ~\ref{tbl:MacroalgaeTable}, table ~\ref{tbl:NetTable}) are normalised to grams of wet weight macroalgal substrate. Due to the difference in colonisable surface area between samples taken from the structurally diverse \emph{Chnoospora} sp., \emph{Padina} sp. and \emph{Sargassum} sp., the potential colonisable space is not comparable. To overcome the difference of available surface area for colonisation between samples, Tester et al. (2014) have proposed an artificial substrate and a standardised sampling method. Further, \emph{Gambierdiscus} spp. show species specific preference for some macroalgae (review by \citep{cruz2006macroalgal}) which may be linked to the macroalgae's chemical defences \cite{parsons2011examination}. The implementation of artificial substrate and unified methodology are essential for comparative studies of \emph{Gambierdiscus} presence in a  given area \cite{tester2014sampling}. The authors of this paper recommend the implementation of the artificial substrate in conjunction with the primers described herein for quantitative monitoring purposes.


%address pos neg part

Patchy distribution of \emph{Gambierdiscus} has been observed previously and is confirmed around Heron Island for \emph{G. lapillus} \cite{taylor1986underwater}. Table ~\ref{tbl:MacroalgaeTable} shows large variation within the spatially replicated sampled for most of the sites. For example, sample 3 consisted of three replicates of \emph{Padina} sp., however on (3B) no \emph{G. lapillus} was detected while (3A) was positive for detection but much lower than (3C).
There is no significant difference in the presence/absence of \emph{G. lapillus} between \emph{Padina} sp., \emph{Saragassum} sp. and \emph{Chnoospora sp.}. For example (6A - \emph{Chnoospora} sp.) and (6B - \emph{Padina} sp.) hosted comparable cell numbers while no \emph{G. lapillus} cells were detected on (6C - \emph{Padina} sp.). While the presence and cell number of \emph{G. lapillus} varied between spatial replicates, only in sample number 25 out of the 33 samples were no \emph{G. lapillus} detected. 
The high variability of presence/absence  within the spatial replicates (Fig. ~\ref{fig:envposneg}) confirms that spatial replication is not an adequate assessment for \emph{Gambierdiscus}. This trend was observed by Taylor et al. (1983) in a cell count comparison of 7 \emph{Bryothamnion} plants spaced 5 to 10 cm apart. They reported variation in cell numbers from 5 to 70 per g of algae \cite{taylor1986underwater}. Biological replicates, taken from the same plant, may be yield more consistency.

%nets
\emph{G. lapillus} was also detected in two of the area net samples (table ~\ref{tbl:NetTable}). Motile behaviour has been observed in the field at various time points \cite{yasumoto1977finding,bomber1987ecology}. Parsons et al. (2011) reported \emph{Gambierdiscus} sp. behaviour as facultative epiphytes during lab scale experiments, as cells showed attachment as well as motile stages over time in the presence of different macroalgae \cite{parsons2011examination}. Taylor \& Gustavson (1983) reported that \emph{Gambierdiscus} cells were captured in plankton tows by de Silva in 1956 but reported as \emph{Goniodoma} \cite{taylor1986underwater}. To the authors best knowledge, this constitutes the first report of capturing free swimming \emph{Gambierdiscus} cells since de Silva. 

%discuss macroalgal sunstrate thing
\emph{Gambierdiscus} spp. display preference for different macroalgae. Parsons et al. (2011) tested the the affinity and survivability of \emph{Gambierdiscus} clone BIG-12 in for 24 macroalgal specimens \cite{parsons2011examination}. In the study, the clone BIG-12 was identified as \emph{G. toxicus} using  the dichotomous species identification key proposed by Litaker et al. (2009) \cite{litaker2009taxonomy}. However without molecular confirmation, this species designation is uncertain. Kohli et al. (2014) visually identified \emph{Gambierdiscus} from Merimbula, Australia, as \emph{G. toxicus} however molecular ID showed that it was \emph{G. carpenteri} instead. Further, morphological intra-species variation has been demosntrated in \emph{Gambierdiscus} \cite{bravo2014cellular,kretzschmar2016characterization,fraga2016gambierdiscus}. The verification of which species of \emph{Gambierdiscus} constituted BIG-12 in the Parsons et al. (2011) as substrate preference differs between \emph{Gambierdiscus} spp. \cite{parsons2011examination}. The macroalgal hosts assessed by Parsons et al (2011) were monitored on the basis of BIG-12 cell attachment, motile cells and cell death over time. They found that attachment and survival of \emph{Gambierdiscus} cells was limited for macroalgae that employ chemical defences against herbivory. This suggests that the macroalgae chosen for screening should be considered for their attributes as a host and likely a range of samples should be screened.
Here here we have demonstrated that \emph{G. lapillus} colonised \emph{Chnoospora} sp., \emph{Padina} sp. and \emph{Sargassum} sp. Further, \emph{G. lapillus} also colonised \emph{Halimeda} sp. \cite{kretzschmar2016characterization}.

% Look at actual cell numbers incl Nishimura cell numbers comparable table 4
The \emph{G. lapillus} cell densities within and between spatial replicates is consistently low (Fig. ~\ref{fig:envHG7}). The average is around 5 cells per g.ww algae (CALCULATE ACTUAL), which is comparable to the cell densities recorded by Nishimura et al. (2016) in their environmental screening (Table 4 in \cite{nishimura2016quantitative}). Heuristics indicate that low cell numbers should correlate to low ciguatera risk, however there are other factors at play. The toxicity of the species is essential in determining risk, and this could be subject to intra-strain variation \cite{kretzschmar2016characterization}. Furthermore, the type of macroalgal substrate is key. Cruz-Rivera et al. (2006) report several of the commonly habitated macroalgae are not palatable for herbivores, which suggests that the protist communities residing on their surface would not enter the food chain. They suggest that chemically and structurally defended macroalge could serve as a refugium for toxic \emph{Gambierdiscus} spp. Conversely, fast growing palatable hosts would serve as a consistent delivery mechanism of toxic \emph{Gambierdiscus} spp. for bioaccumulation. Low cell numbers in this scenario could still serve as the basis for a highly ciguateric trophic system, depending on the speed of uptake \cite{cruz2006macroalgal}.
The brown macroalgae collected for this study vary in palatability and defence mechanisms(Appendix A in \cite{cruz2006macroalgal} and references therein): The palatability of \emph{Chnoospora} spp. is undocumented though quick response to nutrient pulses suggests fast growth rate; \emph{Padina} spp. vary in palatability depending on species in question and the herbivore; and \emph{Sargassum} spp. show high intra-species variability in palatability. \emph{G. lapillus} has also been isolated from  \emph{Halimeda}, a green algae with extensive chemical and structural defences which is rarely consumed by herbivores \cite{kretzschmar2016characterization,cruz2006macroalgal}.

%results re poly
While \emph{G. polynesiensis} was not detected in any of the environmental samples, the qPCR primer set described here is sufficiently sensitive and specific to be utilized for monitoring programs. \emph{Gambierdiscus} distribution is patchy and colonisation is governed by species specific preference for substrate. To the best of the authors' knowledge the preference of \emph{G. polynesiensis} for colonising \emph{Chnoospora} sp., \emph{Padina} sp. and \emph{Sargassum} sp. is not known. Hence it is possible \emph{G. polynesiensis} is present at other sites along the GBR, or even at Heron Island on different macroalgae.

\newpage
\section{Conclusion}
%re-state aim w/ sensitivity etc, tool fort env monitoring, detection limits
%With the description of these primer sets, four of the five Australian \emph{Gambierdiscus} spp. can be monitored, sans \emph{G. toxicus}
\section{Acknowledgements}
Gratitude to Dr. Adachi and Dr. Nishimura for supplying \emph{G. scabrosus} as well as Dr. Kirsty Smith and Dr. Lesley Rhodes for supplying \emph{G. cheloniae}, both species were used for DNA for cross reactivity assessment. Thanks to Bojana M? for Matlab assistance.
\FloatBarrier
\newpage
\bibliographystyle{acm}
\bibliography{references.bib}
\end{document}