\documentclass[12pt]{article}
\usepackage[hcentering,bindingoffset=20mm]{geometry}
\usepackage{placeins}
\usepackage[numbib]{tocbibind}
\usepackage{rotating}
\usepackage[square,sort,comma,numbers]{natbib}
\usepackage{graphicx}
\usepackage{tabularx}
\linespread{1.3}
\usepackage{gensymb}
\usepackage{longtable}
\usepackage{lscape}
\usepackage{url}
\addtolength{\textwidth}{2cm}
\addtolength{\hoffset}{-1cm}
\addtolength{\textheight}{2cm}
\addtolength{\voffset}{-1cm}
\setlength{\parindent}{0pt}
\title{Development of semi-quantitative PCR assays for the detection and enumeration of \emph{Gambierdiscus lapillus} and \emph{Gambierdiscus polynesiensis} (Gonyaulacales, Dinophyces) at the Great Barrier Reef, Australia.}
\author{me}
\date{}
\begin{document}
\maketitle
\paragraph{}Anna Liza Kretzschmar\\
Plant Functional Ecology and Climate Change Cluster (C3), University of Technology Sydney, Ultimo, 2007 NSW, Australia, anna.kretzschmar@uts.edu.au
\paragraph{}Arjun Verma \\
Plant Functional Ecology and Climate Change Cluster (C3), University of Technology Sydney, Ultimo, 2007 NSW, Australia
\paragraph{}Gurjeet Kohli\\
Plant Functional Ecology and Climate Change Cluster (C3), University of Technology Sydney, Ultimo, 2007 NSW, Australia
\paragraph{}Shauna Murray\\
Plant Functional Ecology and Climate Change Cluster (C3), University of Technology Sydney, Ultimo, 2007 NSW, Australia
\newpage
\section{Abstract}
Ciguatera fish poisoning is a potentially fatal illness contracted through ingestion of seafood containing analogues of ciguatoxins. It is prevalent in tropical regions worldwide, including in Australia. Ciguatoxins are produced by some species of the genus \emph{Gambierdiscus}. Rapid \emph{Gambierdiscus} species identification through quantitative PCR is part of UNESCO's global ciguatera strategy, along with the identification of toxicity of species. In this study we developed a rapid qPCR assay to quantify the species Gambierdiscus lapillus, which is found in Great Barrier Reef region of northern Australia, and produces an uncharacterized analogue of maitotoxin, and also possible ciguatoxin-like compounds. Here, we describe the specificity and efficiency of the method. We tested this assay on samples from 33 sites around Heron Island in the southern Great Barrier Reef. We explore the distribution of \emph{G. lapillus} across spatial replicates and three marcoalgal hosts. 


\newpage
\section{Introduction}
%The genus Gambierdiscus and its morphological diversity, and why qPCR is needed to detect species.
Benthic dinoflagellates of the genus \emph{Gambierdiscus} Adachi \& Fukuyo produce neurotoxic ciguatoxins (CTX) which bioaccumulate in fish and other marine life, causing ciguatera fish poisoning (CFP) \citep{chinain1997intraspecific,holmes1998gambierdiscus}. The genus was discovered in 1977 with \emph{G. toxicus} Adachi \& Fukuyo \citep{adachi1979thecal} and the taxon was considered monotypic till 1995, when \emph{G. belizeanus} Faust was discovered \citep{faust1995observation}. Since then the genus has expanded to 13 described species and 6 ribo/species types, with new species discovered this year \citep{fraga2016gambierdiscus,litaker2010global,adachi1979thecal,faust1995observation,chinain1999morphology,litaker2009taxonomy,nishimura2014morphology,fraga2011gambierdiscus,xu2014distribution,fraga2014genus} (ALK in prep).
There are still as yet undescribed \textit{Gambierdiscus} species reported,  so reports of species prior to 2009, when the first major revision of the genus was undertaken, may have different species names and far lower levels of species discrimination than is currently accepted \citep{berdalet2012global,nishimura2014morphology}. As many species are highly morphologically similar to one another, molecular genetic tools are essential to determine the distribution and abundance of  \textit{Gambierdiscus} species  and ciguatera risk \citep{kohli2014high} (ALK in prep) . For this reason, the development of molecular tools for distinguishing \textit{Gambierdiscus} species has been prioritized in the proposed global ciguatera strategy by United Nations Educational Scientific and Cultural Organization's (UNESCO) international panel for harmful algal blooms \citep{litaker2010global,globalcig}. \\

%Toxins of Gambierdiscus. How to measure. Which species produce which toxins. Why this is important to know.

\emph{Gambierdiscus} spp. produce a suite of different polyketide toxins - CTX, maitotoxin (MTX), gambierone, gambieric acid and gambierol have been characterised to date \citep{satake1993gambierol,nagai1992gambieric,rodriguez2015gambierone,murata1993structure,murata1989structures}. While any of these can contribute to toxicity, only CTX has been clearly found to cause CFP \citep{chinain1997intraspecific,holmes1998gambierdiscus}. It is not yet well understood which \textit{Gambierdiscus} species produce CTXs, and many different assays have been used to determine CTX toxicity \citep{globalcig}. Bioassays, such as mouse bioassays and neuroblastoma cell-line bioassays are good indicators of the toxicity of an organism, however species/strain specific toxin profiles needs to be elucidated with LC-MS \citep{diogened2014chemistry}. \textit{Gambierdiscus polynesiensis} Chinain \& Faust is the only species whose production of CTX congeners has been consistently verified by LC-MS \citep{chinain2010growth}. The production of CTX-like compounds \emph{Gambierdiscus lapillus} is currently being investigated, as an uncharacterised CTX-like peak has been detected using LC-MS/MS (ALK in prep). Determining the toxin profile for species of \textit{Gambierdiscus} requires toxin standards for comparative peak analysis; however these are currently not commercially available. Therefore, progress in determining the toxins produced by species of \emph{Gambierdiscus} has been comparatively slow.\\

%CFP 
The diagnosis and report of CFP is difficult due to the breadth of gastrointestinal, cardiovascular and neurological presentation of CFP and relies in specific training of the physician \citep{sims1987theoretical}. 
To prevent outbreaks, monitoring programs are of essence. CFP was determined  to be  a neglected tropical disease by a panel of experts co-ordinated by the UNESCO, and a global ciguatera strategy was developed \cite{globalcig}. The global ciguatera strategy involves investigating species of the genus \emph{Gambierdiscus}, elucidating which species produce CTX through LC-MS and developing efficient and reliable monitoring tools for the species of interest \citep{globalcig}. The identification of \emph{Gambierdiscus} spp. based on morphological features, using light and electron microscopy, is difficult as species can be highly similar despite significant genetic differences \citep{kohli2014high,bravo2014cellular}, (ALK in prep). For this reason, quantitative PCR (qPCR) has been suggested as a method to identify species as part of the global ciguatera strategy \citep{globalcig}. \\

%Which qPCR assays have been developed to date, why more are necessary
For qPCR based species identification of \emph{Gambierdiscus}, assays have been developed either using SYBR Green of TaqMan probes \cite{vandersea2012development,nishimura2016quantitative}. The former relies on target specific primer design and flourometric meassurement of the chelating agent SYBR Green, which is directly proportional to the amount of PCR product present after each cycle. The latter relies on the release of a flourophore upon incoropration of a target specific probe, indexed to the PCR product present per cycle \citep{smith2009advantages}. Currently the qPCR assays available for \emph{Gambierdiscus} identification are as follows: \emph{G. belizeanus}, \emph{G. caribaeus} Vandersea, Litaker, Faust, Kibler, Holland \& Tester, \emph{G. carpenteri} Vandersea, Litaker, Faust, Kibler, Holland \& Tester, \emph{G. carolinianus} Vandersea, Litaker, Faust, Kibler, Holland \& Tester, \emph{Gambierdiscus} sp. ribotype 2 and \emph{G. (Fukoyoa) ruetzleri} (Faust, Litaker, Vandersea, Kibler, Holland \& Tester) G\`omez, Qiu, Lopes \& Lin utilising SYBR Green \citep{vandersea2012development}. Nishimura et al. (2016) developed TaqMan probes for \emph{G. australes} Chinian \& Faust, \emph{G. scabrosus} Nishimura, Sato \& Adachi, \emph{Gambierdiscus} sp. type 2, \emph{Gambierdiscus} sp. type 3 and \emph{G. (Fukoyoa)} cf. \emph{yasumotoi} (Holmes) G\`omez, Qiu, Lopes \& Lin \citep{nishimura2016quantitative}. This constitutes 6 out of 13 verified species specific qPCR primer sets for described \emph{Gambierdiscus} spp. and 3 out of 6 undescribed \emph{Gambierdiscus} sp. types/ribotypes , as well as 2 out of 3 qPCR primer sets for species of the genus \emph{Fukoyoa} (seceded from the genus \emph{Gambierdiscus} in 2015 \citep{gomez2015fukuyoa}) \textit{Fukoyoa} spp. are of interest as MTX producers, as the involvement of that toxin in CFP has not been resolved \citep{kohli2014feeding}. \\

%Australia
Australia has CFP endemic areas, most notably the Great Barrier Reef, yet the cases are highly under reported with a predicted rate of 20\% \citep{lewis2006ciguatera}. The country's coastline is expansive and sites sampled for the presence of \emph{Gambierdiscus} spp. are sparse (ALK in prep). The 4 species of \emph{Gambierdiscus} that have been identified from Queensland and New South Wales are as follows: \emph{G. belizeanus} \citep{murray2014molecular}, \emph{G. carpenteri} \citep{kohli2014high}, \emph{G. lapillus} (ALK in prep) and \emph{G. toxicus} \citep{hallegraeff2010algae}, as well as \emph{F. yasumotoi} \citep{murray2014molecular}. Pyrosequencing identified \emph{Gambierdiscus} to the genus level in Western Australia \citep{kohli2014cob} , indicating that this is a coastline that should be examined further for CFP risk. Despite the prevalence of CFP in Australia, a verified CTX producer has not yet been identified.
qPCR primers that can be used for identifying and monitoring Australia have been developed for \emph{G. belizeanus}, \emph{G. carpenteri} and \emph{F. yasumotoi} \citep{nishimura2016quantitative,vandersea2012development}. \\ 

%aim
The aim of this study was to develop qPCR primers assays to detect the two species \emph{G. lapillus} and \emph{G. polynesiensis}. The latter is a species found in the Pacific Island nations that produces a suite of CTXs, identified by LC-MS, and would be a plausible causative agent of CFP in Australia. The former is a recently identified species from the Great Barrier Reef whose toxin profile did not include any of the known CTXs but displayed an unusual peak in the CTX phase of the LC-MS which makes it a potentially important candidate to monitor. Neuroblastoma cell assay was conducted to investigate CTX specific toxicity and draw inferences for environmental relevance.
\newpage
\section{Materials and methods}
\subsection{Clonal strains and culturing conditions}
Three strains of \emph{G. lapillus} and two strains of \emph{G. polynesiensis} were isolated from Heron Island, Australia, and Rarotonga, Cook Islands, respectively (Table ~\ref{tbl:StrainTable}). The cultures were maintained F$\bullet$-10 medium at 27 $^{\circ}$C, 60mol$\bullet$-m$^{2}$ $\bullet$-s light in 12hr:12hr light to dark cycles.
\FloatBarrier
\begin{table}
\caption{List of \emph{Gambierdiscus} clonal strains used for the qPCR assay.}
\label{tbl:StrainTable}
\begin{tabular}{ | p{2cm} | p{2cm} | p{2cm}| p{3cm} | p{3cm} | p{2cm} | }
\hline
\textbf{Species} & \textbf{Collection site} & \textbf{Collection date} &\textbf{Latitude} & \textbf{Longitude} & \textbf{Strain name} \\
\hline
\emph{G. lapillus} &Heron Island, Australia &July 2014 &23$^{\circ}$ 4420' S&151$^{\circ}$ 9140' E & HG4 \\
\hline
&&&&& HG6\\
\hline
&&&& &HG7\\
\hline
\emph{G. polynesiensis}&Rarotonga, Cook Islands&November 2014 &21$^{\circ}$ 2486' S&159$^{\circ}$ 7286' W & CG14 \\
\hline
&&&&&CG15\\
\hline
\end{tabular}
\end{table}
\subsection{DNA extraction and species specific primer design}
Genomic DNA was extracted using the CTAB method \citep{zhou1999analysis}. Purity and concentration of the extract was measured by Nanodrop (Nanodrop2000, Thermo Scientific), the integrity was visualised on 1\% agarose gel.
Unique primer sets were designed for the small-subunit (SSU) rDNA region of \emph{G. lapillus}, as reported by Kretzschmar et al., and \emph{G. polynesiensis}, as available in GenBank reference database. The target sequences were aligned against sequences of all other \emph{Gambierdiscus}spp. from GenBank reference database, with the MUSCLE algorithm (maximum of 8 iterations) \citep{edgar2004muscle} used through the Geneious software, version 8.1.7 \citep{kearse2012geneious}. Unique sites were determined manually (Table ~\ref{tbl:PrimerTable}). Primers were synthesised by Integrated DNA Technologies (IA, USA).
Primer sets were tested systematically for secondary product formation for all 3 strains of \emph{G. lapillus} and 2 strains of \emph{G. polynesiensis} (Table ~\ref{tbl:StrainTable}) via standard PCR in 25$\mu$l mixture in PCR tubes. The mixture contained 0.6 $\mu$M forward and reverse primer, 0.4 $\mu$M BSA, 2 - 20 ng DNA, 12.5 $\mu$l 2xEconoTaq (Lucigen) and 7.5 $\mu$l PCR grade water.
The PCR cycling comprised of an initial 10 min step at 94 $^{\circ}$C, followed by 30 cycles of denaturing at 94 $^{\circ}$C for 30 sec, annealing at 60 $^{\circ}$C for 30 sec and extension at 72 $^{\circ}$C for 1 min, finalised with 3 minutes of extension at 72 $^{\circ}$C. Products were visualised on 1\% agarose gel.
\FloatBarrier
\begin{table}
\caption{List of species specific qPCR primer sets for SSU rDNA.}
\label{tbl:PrimerTable}
\begin{tabular}{ | p{2cm} | p{2cm} | p{2cm} | p{2cm} | p{7cm} | }
\hline
\textbf{DNA target} & \textbf{Amplicon size} & \textbf{Primer name} & \textbf{Synthesis direction of primer} & \textbf{Sequence (5'-3')} \\
\hline
\emph{G. lapillus} & &qGlapSSU2F & Forward & \\
\hline
& &qGlapSSU2R & Reverse & \\
\hline
\emph{G. polynesiensis}& &qGpolySSU2F& Forward & \\
\hline
& &qGpolySSU2R & Reverse & \\
\hline
\end{tabular}
\end{table}
\subsection{Evaluation of primer specificity}
To verify primer set specificity as listed in Table ~\ref{tbl:PrimerTable}, DNA was extracted via CTAB from \emph{G. belizeanus} (CCMP401), \emph{G. australes} (CCMP1650 and CG61) and \emph{G. pacificus} (CAWD149). sPIKEY 8A - \emph{G. xx} (CAWD232) DNA was extracted using a PowerSoilâ„¢ DNA isolation kit (Mo Bio Inc., CA, USA) by Dr. Kirsty Smith, Cawthron Institute, New Zealand. Cross reactivity for \emph{G. scabrosus} () DNA was extracted using DNeasy Plant Mini Kit (Quiagen, Tokyo, Japan) according to the manufacturer's protocol, by Dr. Nishimura, Kochi University, Japan. For all extracted samples, the presence and integrity of genomic DNA was assessed on 1\% agarose gel. Primer sets designed for \emph{G. lapillus} and \emph{G. polynesiensis} were tested for cross-reactivity against all other \emph{Gambierdiscus} spp. available via PCR. PCR amplicons were visually assessed on 1\% agarose gel.
\subsection{Evaluation of primer sensitivity}
The qPCR reaction mixture contained 10 $\mu$l SYBR Select Master Mix (Thermo Fisher Scientific), 7 $\mu$l MilliQ water, 0.5 $\mu$M forward and reverse primers and 2 - 20 ng DNA template, for a final volume of 20 $\mu$l.
\subsection{Calibration curve construction}
Standard curves were constructed via two methods: (1) synthetic gene fragment based (henceforth referred to as gene based calibration) and (2) cell based calibration curves. The former was constructed from a 10-fold serial dilution of a synthesised fragment containing the SSU target sequence, forward and reverse primer sites and 50bp flanking both primer sites matching sequencing results. Cell-based standard curves were constructed using 10-fold dilutions of gDNA extract of known cell concentrations.
\subsubsection{Gene based calibration curve}
A synthetic gBlocks gene fragment spanning the target SSU sequence, primer sites and 50bp on either end was synthesised for both \emph{G. lapillus} and \emph{G. polynesiensis}.
\subsubsection{Cell based calibration curve}
Three cultures of \emph{G. lapillus} and two cultures of \emph{G. polynesiensis} (Table ~\ref{tbl:StrainTable}) were used to construct cell based standard curves. Cells were counted microscopically () with a Sedqick Rafter Counting Chamber. DNA was extracted using xx kit, as per the manufacturer's instructions. The gDNA extracts were 10-fold serial diluted.
%\subsection{Quantification of SSU rDNA copy number per cell of \emph{G. lapullis} and \emph{G. polynesiensis}}
\subsection{Toxicity of \emph{G. lapillus}}
- if FLIPR assays work out
\subsection{Screening environmental samples for \emph{G. lapullis} and \emph{G. polynesiensis} abundance}
Around Heron Island and Heron Reef (Fig. xx) 33 sites were sampled in October 2015, in spatial replicates (A, B, C; ~\ref{tbl:MacroalgaeTable}) within a 2m radius. Macroalgae targeted were \textit{Chnoospora implexa}, \textit{Padina} sp. and \textit{Saragassum} sp. where ~200g were taken, epiphytes were dislodged from macroalagae and preserved in 10ml RNAlater till DNA extraction.
Plankton nets were dragged through the current (samples 34 - 38; ~\ref{tbl:NetTable}) for five minutes, then processed as macroalgal samples.

\newpage
\section{Results}
\subsection{Evaluation of primer specificity}
-picture of gel?
\FloatBarrier
\begin{table}
\caption{Cross-reactivity of qPCR primer sets.}
\label{tbl:CrossreactTable}
\begin{tabular}{ | p{4cm} | p{3cm} | p{2cm} | p{2.5cm} | p{2.5cm} | }
\hline
\textbf{Template} & \textbf{Strain name} & \textbf{gDNA gel band} & \textbf{GlapSSU2F-GlapSSU2R} & \textbf{GpolySSUF-GpolySSUR} \\
\hline
\emph{G. australes} & CCMP1650 &+&-&- \\
\hline
& CG61 &+&-&- \\
\hline
\emph{G. belizeanus}&CCMP401&+&-&-\\
\hline
\emph{G. lapillus}&HG1&+&+&-\\
\hline
&HG4&+&+&-\\
\hline
&HG6&+&+&-\\
\hline
&HG7&+&+&-\\
\hline
&HG26&+&+&-\\
\hline
\emph{G. pacificus}&CAWD149&+&-&-\\
\hline
\emph{G. polynesiensis}&CG14&+&-&+\\
\hline
&CG15&+&-&+\\
\hline
\emph{G. scabrosus}&&+&-&-\\
\hline
\end{tabular}
\end{table}
\FloatBarrier
 
\subsection{Evaluation of primer sensitivity}
-figs qPCR std curves
\FloatBarrier
\begin{figure}
%\includegraphics[scale=.22]{MH_variability.jpg}
\caption{qPCR standard curves of \emph{G. lapillus} strains HG4: (A) HG6: (B) HG7:(C). Error bars represent the deviation of technical replicates during reactions.}
\label{fig:lapiStd}
\end{figure}
\FloatBarrier
\FloatBarrier
\begin{figure}
%\includegraphics[scale=.22]{MH_variability.jpg}
\caption{qPCR standard curves of \emph{G. polynesiensis} strains CG14: (A) CG15: (B). Error bars represent the deviation of technical replicates during reactions.}
\label{fig:polyStd}
\end{figure}
\FloatBarrier
-Ct of known gBlocks gives copy number - extrapolate those to SSU rDNA copies for cell. can compare to
\subsection{Quantification of SSU rDNA copy number per cell of \emph{G. lapullis} and \emph{G. polynesiensis}}
-gblocks calc
\subsection{Screening environmental samples for \emph{G. lapullis} and \emph{G. polynesiensis} abundance}
- map of Heron island and sampling sites
- not sure how to label Table ~\ref{tbl:MacroalgaeTable} biological rep section, will change that table once I get clarity on that.
\FloatBarrier
\begin{longtable}{ | p{1cm} | p{1cm} | p{3cm} | p{4cm} | p{4cm} | }
\caption{Screening of macroalgal samples of \emph{G. lapillus} and \emph{G. polynesiensis}and cell density estimates via qPCR. \emph{G. lapillus} cells numbers were modelled on the type strain HG7. \emph{G. polynesiensis} cell numbers were modelled on the strain CG14. N/D denotes not detected; N/A denotes not attempted due to loss of sample.}\\
\hline
\label{tbl:MacroalgaeTable}
\textbf{Sample ID}&\textbf{Spatial rep}&\textbf{Macroalgal substrate}&\textbf{\textit{G. lapillus} cell}&\textbf{\textit{G. polynesiensis} SSU rDNA copy number }\\
\hline
1&A&\emph{Padina} sp.&N/D&N/D\\
\hline
1&B&\emph{Sargassum} sp.&8.14E+16
&N/D\\
\hline
1&C&\emph{Padina} sp.&2.21E+17
&N/D\\
\hline
2&A&\emph{Padina} sp.&N/D&N/D\\
\hline
2&B&\emph{Padina} sp. \& \emph{C. implexa}&1.40E+17
&N/D\\
\hline
2&C&\emph{Padina} sp.&1.42E+17
&N/D\\
\hline
3&A&\emph{Padina} sp.&9.64E+17
&N/D\\
\hline
3&B&\emph{Padina} sp.&N/D&N/D\\
\hline
3&C&\emph{Padina} sp.&3.06E+17
&N/D\\
\hline
4&A&\emph{Padina} sp. \& \emph{C. implexa}&2.53E+17
&N/D\\
\hline
4&B&\emph{Chnoospora implexa}&2.54E+18
&N/D\\
\hline
4&C&\emph{Padina} sp. \& \emph{C. implexa}&N/D&N/D\\
\hline
5&A&\emph{Padina} sp.&N/D&N/D\\
\hline
5&B&\emph{Padina} sp.&2.23E+17
&N/D\\
\hline
5&C&\emph{Padina} sp.&6.45E+17
&N/D\\
\hline
6&A&\emph{C. implexa}&5.36E+17
&N/D\\
\hline
6&B&\emph{Padina} sp.&5.31E+17
&N/D\\
\hline
6&C&\emph{Padina} sp.&N/D&N/D\\
\hline
7&A&\emph{Padina} sp.&6.39E+16
&N/D\\
\hline
7&B&\emph{Padina} sp.&N/D&N/D\\
\hline
7&C&\emph{Padina} sp.&N/D&N/D\\
\hline
8&A&\emph{C. implexa}&N/D&N/D\\
\hline
8&B&\emph{Padina} sp.&7.15E+17
&N/D\\
\hline
8&C&\emph{Padina} sp. \& \emph{C. implexa}&1.68E+17
&N/D\\
\hline
9&A&\emph{Padina} sp. \& \emph{C. implexa}&N/D&N/D\\
\hline
9&B&\emph{Padina} sp.&3.60E+17
&N/D\\
\hline
9&C&\emph{Padina} sp.&4.41E+17
&N/D\\
\hline
10&A&\emph{Padina} sp. \& \emph{C. implexa}&1.92E+17
&N/D\\
\hline
10&B&\emph{Padina} sp. \& \emph{C. implexa}&2.96E+17
&N/D\\
\hline
10&C&\emph{Padina} sp.&N/D&N/D\\
\hline
11&A&\emph{Padina} sp. \& \emph{Saragassum} sp.&N/D&N/D\\
\hline
11&B&\emph{Padina} sp. \& \emph{Saragassum} sp.&7.51E+18
&N/D\\
\hline
11&C&\emph{Padina} sp. \& \emph{Saragassum} sp.&4.54E+17
&N/D\\
\hline
12&A&\emph{C. implexa}&N/D&N/D\\
\hline
12&B&\emph{C. implexa}&5.67E+17
&N/D\\
\hline
12&C&\emph{C. implexa}&5.65E+17
&N/D\\
\hline
13&A&\emph{C. implexa}&N/D&N/D\\
\hline
13&B&\emph{Padina} sp. \& \emph{C. implexa}&1.28E+18
&N/D\\
\hline
13&C&\emph{Padina} sp. \& \emph{C. implexa}&3.13E+16
&N/D\\
\hline
14&A&\emph{Padina} sp. \& \emph{C. implexa}&8.33E+16
&N/D\\
\hline
14&B&\emph{Padina} sp. \& \emph{C. implexa}&N/D&N/D\\
\hline
14&C&\emph{Chnoospora implexa}&1.72E+18
&N/D\\
\hline
15&A&\emph{Padina} sp. \& \emph{C. implexa}&9.18E+17
&N/D\\
\hline
15&B&\emph{C. implexa}&9.40E+17
&N/D\\
\hline
15&C&\emph{C. implexa}&N/D&N/D\\
\hline
16&A&\emph{C. implexa}&1.12E+18
&N/D\\
\hline
16&B&\emph{C. implexa}&2.32E+17
&N/D\\
\hline
16&C&\emph{C. implexa}&6.18E+17
&N/D\\
\hline
17&A&\emph{C. implexa}&1.23E+17
&N/D\\
\hline
17&B&\emph{C. implexa}&6.04E+17
&N/D\\
\hline
17&C&\emph{Padina} sp. \& \emph{C. implexa}&N/D&N/D\\
\hline
18&A&\emph{C. implexa}&2.58E+17
&N/D\\
\hline
18&B&\emph{C. implexa}&N/D&N/D\\
\hline
18&C&\emph{C. implexa}&2.55E+19
&N/D\\
\hline
19&A&\emph{C. implexa}&6.46E+16
&N/D\\
\hline
19&B&\emph{Padina} sp. \& \emph{C. implexa}&4.61E+17
&N/D\\
\hline
19&C&\emph{Padina} sp. \& \emph{C. implexa}&2.00E+17
&N/D\\
\hline
20&A&\emph{Padina} sp. \& \emph{C. implexa}&8.96E+17
&N/D\\
\hline
20&B&\emph{Padina} sp. \& \emph{C. implexa}&5.34E+17
&N/D\\
\hline
20&C&\emph{C. implexa}&N/D&N/D\\
\hline
21&A&\emph{Padina} sp. \& \emph{C. implexa}&1.91E+17
&N/D\\
\hline
21&B&\emph{C. implexa}&3.57E+17
&N/D\\
\hline
21&C&\emph{Padina} sp.&1.60E+18
&N/D\\
\hline
22&A&\emph{C. implexa}&N/D&N/D\\
\hline
22&B&\emph{C. implexa}&5.15E+17
&N/D\\
\hline
22&C&\emph{C. implexa}&1.44E+17
&N/D\\
\hline
23&A&\emph{C. implexa}&2.79E+17
&N/D\\
\hline
23&B&\emph{C. implexa}&N/D&N/D\\
\hline
23&C&\emph{C. implexa}&8.37E+17
&N/D\\
\hline
24&A&\emph{Padina} sp. \& \emph{C. implexa}&2.70E+17
&N/D\\
\hline
24&B&\emph{Padina} sp. \& \emph{C. implexa}&2.16E+18
&N/D\\
\hline
24&C&\emph{Padina} sp.&1.05E+18
&N/D\\
\hline
25&A&\emph{Padina} sp.&N/D&N/D\\
\hline
25&B&\emph{Padina} sp.&N/D&N/D\\
\hline
25&C&\emph{Padina} sp.&N/D&N/D\\
\hline
26&A&\emph{Saragassum} sp.&N/D&N/D\\
\hline
26&B&\emph{Saragassum} sp.&3.24E+17
&N/D\\
\hline
26&C&\emph{Saragassum} sp.&7.80E+18
&N/D\\
\hline
27&A&\emph{Saragassum} sp.&N/D&N/D\\
\hline
27&B&\emph{Saragassum} sp.&3.46E+18
&N/D\\
\hline
27&C&\emph{Saragassum} sp.&7.38E+17
&N/D\\
\hline
28&A&\emph{Padina} sp.&5.82E+17
&N/D\\
\hline
28&B&\emph{Padina} sp.&2.12E+17
&N/D\\
\hline
28&C&\emph{Padina} sp.&7.37E+17
&N/D\\
\hline
29&A&\emph{C. implexa}&6.32E+17
&N/D\\
\hline
29&B&\emph{C. implexa}&4.91E+17
&N/D\\
\hline
29&C&\emph{C. implexa}&N/D&N/D\\
\hline
30&A&\emph{Padina} sp. \& \emph{Saragassum} sp.&4.00E+17
&N/D\\
\hline
30&B&\emph{Padina} sp. \& \emph{Saragassum} sp.&3.80E+17
&N/D\\
\hline
30&C&\emph{Padina} sp. \& \emph{Saragassum} sp.&N/D&N/D\\
\hline
31&A&\emph{Saragassum} sp.&3.39E+17
&N/D\\
\hline
31&B&\emph{Saragassum} sp.&4.58E+17
&N/D\\
\hline
31&C&\emph{Saragassum} sp.&5.17E+17
&N/D\\
\hline
32&A&\emph{Padina} sp. \& \emph{Saragassum} sp.&1.53E+17
&N/D\\
\hline
32&B&\emph{C. implexa} \& \emph{Saragassum} sp.&3.90E+17
&N/D\\
\hline
32&C&\emph{Padina} sp. \& \emph{Saragassum} sp.&8.78E+17
&N/D\\
\hline
33&A&\emph{Padina} sp.&5.75E+17
&N/D\\
\hline
33&B&\emph{Padina} sp. \& \emph{C. implexa}&2.51E+17
&N/D\\
\hline
33&C&\emph{Padina} sp. \& \emph{C. implexa}&2.98E+18
&N/D\\
\hline
\end{longtable}
\FloatBarrier
\FloatBarrier
\begin{table}
\caption{Screening of tow net samples for free floating \emph{G. lapillus} and \emph{G. polynesiensis}and cell density estimates via qPCR.}
\label{tbl:NetTable}
\begin{tabular}{ | p{4cm} | p{4cm} |p{4cm} | p{4cm} | }
\hline
\textbf{Sample ID}&\textbf{Location}&\textbf{\emph{G. lapillus}cell density}&\textbf{\emph{G. polynesiensis} cell density. N/D denotes not detected.}\\
\hline
34&Shark Bay A?&N/D&N/D\\
\hline
35&Shark bay B&N/D&N/D\\
\hline
36&Shark Bay D&3.59E+17
&N/D\\
\hline
37&North beach D&3.03E+17
&N/D\\
\hline
38&Jetty&1.40E+17
&N/D\\
\hline
\end{tabular}
\end{table}
\FloatBarrier
\newpage
\section{Discussion}

Patchy distribution of \emph{Gambierdiscus} has been observed previously and is confirmed around Heron Island for \emph{G. lapillus}. Table ~\ref{tbl:MacroalgaeTable} shows large variation within the spatial replicated sampled for most of the sites. For example sample 3 consisted of three replicates of \emph{Padina} sp., however on (3B) no \emph{G. lapillus} was detected while (3A) was positive for detection but much lower than (3C).
There is no significant difference in the presence/absence of \emph{G. lapillus} between \emph{Padina} sp., \emph{Saragassum} sp. and \emph{C. implexa}. For example (6A - \emph{C. implexa}) and (6B - \emph{Padina} sp.) hosted comparable cell numbers while no \emph{G. lapillus} cells were detected on (6C - \emph{Padina} sp.). While the presence and cell number of \emph{G. lapillus} varied between spatial replicates, only in sample number 25 out of the 33 samples were no \emph{G. lapillus} detected. \emph{G. lapillus} was also detected in two of the area net samples (table ~\ref{tbl:NetTable}0 in concentrations comparable to the macroalgal colonisation.

While \emph{G. polynesiensis} was not detected in any of the samples, the qPCR primer set described here is sufficiently sensitive and specific to be utilized for monitoring programs.

The results for \emph{G. lapillus} of this study were termed semi-quantitative as the environmental cell numbers detected (table ~\ref{tbl:MacroalgaeTable}, table ~\ref{tbl:NetTable})  due to the difference in surface area of macroalgal substrate between the samples, and hence colonization space. To overcome the difference of available surface area for colonisation between samples, Tester et al. (2014) have proposed an artificial substrate and a standardised sampling method. The implementation is essential for comparative studies of \emph{Gambierdiscus} presence in a  given area \cite{tester2014sampling}. The authors of this paper recommend the implementation of the artificial substrate in conjunction with the primers described herein for monitoring purposes.

%\emph{G. lapillus} was detected on all three macroalgal genera tested (\emph{Chnoospora} sp., \emph{Padina} sp., \emph{Sargassum} sp.). 
\emph{Gambierdiscus} spp. display preference for different macroalgae. Parsons et al. (2012) tested the 
Hence here we have demonstrated that \emph{G. lapillus} colonised \emph{Chnoospora} sp., \emph{Padina} sp. and \emph{Sargassum} sp. Further, \emph{G. lapillus} also colonised \emph{Halimeda} sp. (ALK in prep).


- RE normalization issue, how kits should be used in future to ensure standardized extraction efficiency\\
- need to record surface area of macroalgae for quantitative qPCR Tester et al paper on artificial substrate for uniform detection. Also is there a link between which macroalgal substrate, it's usual surface area and amount of lapillus detected?\\
- difference in abundance even in bio replicates, discuss changes in population even over short spacial distance\\
- is there a difference in cell abundance betwen sides of island, ie reef part or close to open ocean\\
-SSU copies in lapillus and polynesiensis by GBlocks comparison, is there any intra strain variability? If so how does this impact qPCR cell enumeration for monitoring\\
- FLIPR assay, ie is it giving results that match LC-MS for polynesiensis and what does CTX for lapillus look like?
\newpage
\section{Conclusion}
\section{Acknowledgements}
Gratitude to Dr. Adachi and Dr. Nishimura for supplying \emph{G. scabrosus} as well as Dr. Kirsty Smith and Dr. Lesley Rhodes for supplying \emph{G. xx}, both species were used for DNA for cross reactivity assessment. Thanks to Bojana M? for Matlab assistance.
\FloatBarrier
\newpage
\bibliographystyle{acm}
\bibliography{references.bib}
\end{document}