\documentclass[12pt]{article}
\usepackage[hcentering,bindingoffset=20mm]{geometry}
\usepackage{placeins}
\usepackage[numbib]{tocbibind}
\usepackage{rotating}
\usepackage[square,sort,comma,numbers]{natbib}
\usepackage{graphicx}
\usepackage{tabularx}
\linespread{1.3}
%\fontsize{8cm}{1.3em}\selectfont
\usepackage{gensymb}
\usepackage{longtable}
\usepackage{lscape}

\addtolength{\textwidth}{2cm}
\addtolength{\hoffset}{-1cm}


\addtolength{\textheight}{2cm}
\addtolength{\voffset}{-1cm}
\setlength{\parindent}{0pt}

\title{\textbf{ Morphology and phylogenetics of\emph{Gambierdiscus xx} sp. nov. (Gonyaulacales): a new benthic dinoflagellate from the ciguateric Great Barrier Reef (Australia)}}
\author{Anna Liza Kretzschmar, Tim Harwood, Mona Hoppenrath, Shauna Murray}
\date{}

\begin{document}
\maketitle

\section{Abstract}

\section{Introduction}

\section{Materials and methods}

\subsection{Study area}
The specimens were collected from Heron Island, as part of the Great Barrier Reef (GBR). The majority of Australian CFP have been reported from the Great Barrier Reef area, but the causative agent has not yet been identified \cite{lewis2006ciguatera}. Heron Island is located in the Great Barrier Reef and was chosen as a sample site to search for toxic \emph{Gambierdiscus}.\\

 need long and lat CFP is endemic to the GBR area (ref - skinner?). water temp, salinity, shallow waters.

\subsection{Specimen collection}
 
Macroalgal samples were taken around the Heron Island research station. The samples were shaken in seawater to dislodge protists, the samples was then concentrated by filtration. Single cells were isolated from concentrated samples.\\
HG1 was isolated in October 2013 from \emph{Halimedia} sp. The average sea surface temperature was 23.8 $^{\circ}$C. \\
HG4, HG5, HG6, HG7 and HG 26 were isolated in July 2014 from \emph{Padina} sp. The average sea surface temperature was 19.8 $^{\circ}$C.

(table ~\ref{tbl:StrainTable})

\FloatBarrier
\begin{table}
\caption{Collection details for \emph{Gambierdiscus} strains collected from Heron Island, Australia, including morphological features.}
\label{tbl:StrainTable}
\begin{tabular}{ | p{2cm} | p{1.8cm} | p{3cm} | p{2.5cm} | p{1.8cm} | p{4cm} | }
\hline
\textbf{Taxa} & \textbf{Strain} & \textbf{}  & \textbf{} \\
 \hline
\emph{G. xx} & HG1 & &   \\

 & HG4  &  &  \\

 & HG6  &  &   \\

&  HG7  &  &   \\

&  HG26  &  &   \\
  \hline
\emph{G. silvae}& HG5  & &   \\
  \hline
\end{tabular}
\end{table}
\FloatBarrier

\subsection{Culture conditions}
Isolates were separated by single cell isolation and grown as clonal cultures in F/10 medium. The cultures were maintained in F/10 medium at 26 $^{\circ}$C, 60mol/ m$^{2}$/ s light in 12hr:12hr light to dark cycles.

\subsection{Light microscopy}
Sedrick Rafter chamber, Lugols?

\subsection{Scanning electron microscopy}

\subsection{Genomic DNA extraction}
Genomic DNA was extracted using the CTAB method \cite{zhou1999analysis}. Purity and concentration of the extract was measured by Nanodrop, the integrity was visualised on 1\% agarose gel.

\subsection{PCR amplification and sequencing}
Extracted DNA was used as a template to amplify rRDNA sequences in 25$\mu$l mixture in PCR tubes. The mixture contains 0.6 $\mu$M forward and reverse primer, 0.4 $\mu$M BSA, 2 - 20 ng DNA, 12.5 $\mu$l 2xEconoTaq (Lucigen) and 7.5 $\mu$l PCR grade water.\\
The PCR cycling comprised of an initial 10 min step at 94
The LSU D1-D3  and D8-D10 regions as well as the SSU region were amplified with the D1R-F-, FD8-RB and 18ScomF1-18ScomR1 primer sets respectively (table ~\ref{tbl:PrimerTable}).\\
Sanger sequencing was conducted by Macrogen (Korea).
\FloatBarrier
\begin{table}
\caption{List of primers used for phylogenetic analysis of \emph{Gambierdiscus} strains.}
\label{tbl:PrimerTable}
\begin{tabular}{  | p{2cm} | p{7.5cm} | p{2.5cm} | p{2cm} | }
\hline
\textbf{Name} & \textbf{Sequence} & \textbf{Purpose} & \textbf{Reference} \\
\hline
    \multicolumn{4}{| c |}{\textbf{LSU D8-D10 region}}\\
    \hline
   FD8   & GGATTGGCTCTGAGGGTTGGG & Amplification \& sequencing & \cite{chinain1999morphology} \\
   \hline
 GLD8\_421F   & ACAGCCAAGGGAACGGGCTT & Sequencing & \cite{nishimura2013genetic} \\
 \hline
 GLD8\_677R   & TGTGCCGCCCCAGCCAAACT & Sequencing & \cite{nishimura2013genetic} \\
 \hline
   RB   & GATAGGAAGAGCCGACATCGA & Amplification \& sequencing &\cite{chinain1999morphology}  \\
    \hline
  \multicolumn{4}{| c |}{\textbf{SSU region}}\\
    \hline
 18ScomF1 & GCTTGTCTCAAAGATTAAGCCATGC & Amplification \& sequencing & \cite{zhang2005phylogeny} \\
 \hline
 18ScomR1  & CACCTACGGAAACCTTGTTACGAC & Amplification \& sequencing &  \cite{zhang2005phylogeny}  \\
 \hline
 Dino18SF2  & ATTAATAGGGATAGTTGGGGGC & Sequencing &  \cite{zhang2008mitochondrial}\\
 \hline
 Dino18SR1    & GAGCCAGATRCDCACCCA & Sequencing &  \cite{zhang2008mitochondrial}\\ 
 \hline
G10'F    & TGGAGGGCAAGTCTGGTG & Sequencing & \cite{nishimura2013genetic} \\
\hline
G18'R    & GCATCACAGACCTGTTATTG & Sequencing &  \cite{litaker2005reclassification} \\
 \hline
\end{tabular}
\end{table}
\FloatBarrier

\subsection{Sequence alignment and phylogenetic analysis}
The rRNA sequences were aligned using MUSCLE \cite{edgar2004muscle} against publicly available sequences from the NCBI database with a maximum of 8 iterations. The alignment was manually screened and adjusted. The 5' and 3' ends were truncated to render all sequences of the same length leaving D1-D3, D10-D8 and SSU. Outliers for LSU and SSU alignments were \emph{Alexandrium tamarense} AY831407.1 or \emph{Prorocentrum lima} AB189780.1.
PHYML was used for Maximum Likelihood (ML) tree generation with 1000 bootstraps in the general-time-reversal model. \\

\section{Results}

\subsection{\emph{Gambierdiscus xx}}

\subsubsection{Morphology}
Light and SEM, which nomenclature used?

\subsubsection{Phylogenetics}

\begin{figure} 
\includegraphics[scale=.4]{HG_D8D10_phylo.jpg} 
\caption{Maximum Likelihood phylogeny of Heron Island \textit{Gambierdiscus} isolates using 791nt long sequences amplified from the D8D10 LSU region.} 
\label{fig:HGD8D10}
\end{figure} 
\FloatBarrier 
\newpage

\begin{figure} 
\includegraphics[scale=.4]{HG_SSU_phylo.jpg} 
\caption{Maximum Likelihood phylogeny of Heron Island \textit{Gambierdiscus} isolates using 1733nt long sequences amplified from the SSU region.}
\label{fig:HGSSU} 
\end{figure} 
\FloatBarrier 
\newpage

The D8D10 LSU (fig. ~\ref{fig:HGD8D10}) and SSU (fig. ~\ref{fig:HGSSU}) phylogenies of the Heron Island isolates HG1, HG4, HG6, HG7 and HG26 show that these strains fall into a new, well supported clade.\\
In the D8D10 LSU phylogeny (fig. ~\ref{fig:HGD8D10}), HG5 comes out between \emph{G. silvae} and the not yet characterised \emph{Gambierdiscus} sp. 4. whereas in the SSU phylogeny (fig. ~\ref{fig:HGSSU}) HG5 comes out next to \emph{G. polynesiensis}. SSU sequences for \emph{G. silvae} and \emph{Gambierdiscus} sp. 4 are not available from NCBI (table ~\ref{tbl:MorphTable}), hence it is not possible to determine whether HG5 is actually as closely related to \emph{G. polynesiensis} as this phylogeny suggests.\\

\subsubsection{Toxin profile}
Toxin profile analysis for HG4, HG5, HG6 and HG7 was conducted by Tim Harwood at the Cawthron Institute, Nelson, New Zealand, via LC-MS. The protocol is set up to monitor for CTX-3B, CTX-3C, CTX-4A, CTX-4B, MTX-1 and MTX-3. Preliminary results show that solely MTX-3 was detected in all samples. Dr. Harwood reported unusual peaks in some samples, which are subject to further investigation. Whether these peaks are MTX-like or CTX-like is unknown.\\
Toxicity of HG4, HG6 and HG7 is currently under investigation via MBA by i. p. injection by Rex Munday at the Cawthron Institute, Nelson, New Zealand.\\

\section{Discussion}

%\subsection{Morphology}

%\subsection{Phylogeny}

%\subsection{Toxicity}

\section{Acknowledgements}

\section{References}

\newpage
\bibliographystyle{acm}
\bibliography{references.bib}


\end{document}